examples/spark2/src/main/scala/org/apache/carbondata/examples/DataUpdateDeleteExample.scala

```diff
@@ -0,0 +1,173 @@
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.carbondata.examples
+
+import java.io.File
+import java.text.SimpleDateFormat
+
+import org.apache.spark.sql.SaveMode
+import org.apache.spark.sql.SparkSession
+
+import org.apache.carbondata.core.constants.CarbonCommonConstants
+import org.apache.carbondata.core.util.CarbonProperties
+
+object DataUpdateDeleteExample {
+
+  def main(args: Array[String]) {
+
+    // for local files
+    val rootPath = new File(this.getClass.getResource("/").getPath
+      + "../../../..").getCanonicalPath
+    // for hdfs files
+    // var rootPath = "hdfs://hdfs-host/carbon"
+
+    var storeLocation = s"$rootPath/examples/spark2/target/store"
+    var warehouse = s"$rootPath/examples/spark2/target/warehouse"
+    var metastoredb = s"$rootPath/examples/spark2/target"
+
+    import org.apache.spark.sql.CarbonSession._
+    val spark = SparkSession
+      .builder()
+      .master("local")
+      .appName("DataUpdateDeleteExample")
+      .config("spark.sql.warehouse.dir", warehouse)
+      .config("spark.driver.host", "localhost")
+      .config("spark.sql.crossJoin.enabled", "true")
+      .getOrCreateCarbonSession(storeLocation, metastoredb)
+    spark.sparkContext.setLogLevel("WARN")
+
+    // Specify date format based on raw data
+    CarbonProperties.getInstance()
+      .addProperty(CarbonCommonConstants.CARBON_DATE_FORMAT, "yyyy-MM-dd")
+
+    import spark.implicits._
+    // Drop table
+    spark.sql("DROP TABLE IF EXISTS t3")
+    spark.sql("DROP TABLE IF EXISTS t5")
+
+    // use code to create table t5 and insert data
+    var sdf = new SimpleDateFormat("yyyy-MM-dd")
+    var df = spark.sparkContext.parallelize(1 to 10)
+      .map(x => (x, new java.sql.Date(sdf.parse("2015-07-" + (x % 10 + 10)).getTime),
+        "china", "aaa" + x, "phone" + 555 * x, "ASD" + (60000 + x), 14999 + x))
+      .toDF("ID", "date", "country", "name", "phonetype", "serialname", "salary")
+    df.write
+      .format("carbondata")
+      .option("tableName", "t3")
+      .option("tempCSV", "true")
+      .option("compress", "true")
+      .mode(SaveMode.Overwrite)
+      .save()
+
+    sdf = new SimpleDateFormat("yyyy-MM-dd")
+    df = spark.sparkContext.parallelize(1 to 10)
+      .map(x => (x, new java.sql.Date(sdf.parse("2017-07-" + (x % 20 + 1)).getTime),
+        "usa", "bbb" + x, "phone" + 100 * x, "ASD" + (1000 * x - x), 25000 + x))
+      .toDF("ID", "date", "country", "name", "phonetype", "serialname", "salary")
```

Comment:
To easier identify data from different table, please change as below : 
toDF("t5.ID", "t5.date", "t5.country", "t5.name", "t5.phonetype", "t5.serialname", "t5.salary")